pom.xml

```diff
@@ -176,35 +196,101 @@
           </execution>
         </executions>
       </plugin>
-
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-antrun-plugin</artifactId>
-        <!--
-          Fedora-based systems use a different soname for OpenSSL than other linux distributions.
-          Use a custom classifier ending in "-fedora" when building on fedora-based systems.
-        -->
-        <executions>
-          <execution>
-            <phase>initialize</phase>
-            <configuration>
-              <exportAntProperties>true</exportAntProperties>
-              <target>
-                <condition property="classifier_to_use" value="${os.detected.classifier}-fedora" else="${os.detected.classifier}">
-                  <isset property="os.detected.release.like.fedora" />
-                </condition>
-              </target>
-            </configuration>
-            <goals>
-              <goal>run</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
     </plugins>
   </build>
 
   <profiles>
+    <!-- Build distribution statically linked to BoringSSL -->
+    <profile>
+      <id>boringssl</id>
+
+      <properties>
+        <artifactIdToUse>netty-tcnative-boringssl</artifactIdToUse>
+        <boringSslHome>${project.build.directory}/boringssl</boringSslHome>
+        <checkoutDirectory>${boringSslHome}</checkoutDirectory>
+        <boringSslBuildDir>${boringSslHome}/build</boringSslBuildDir>
+        <boringSslVersion>7b8b9c17db93ea5287575b437c77fb36eeb81b31</boringSslVersion>
```

Comment:
@agl `chromium-stable` SGTM.  If you're ok with that, I'll hold off on this PR for now, until you've created the branch.